common4j/src/main/com/microsoft/identity/common/java/providers/nativeauth/NativeAuthResponseHandler.kt

```diff
@@ -48,6 +51,124 @@ class NativeAuthResponseHandler {
 
     private val TAG = NativeAuthResponseHandler::class.java.simpleName
 
+    //region /signup/start
+    /**
+     * Converts the HTTP response for /signup/start API to [SignUpStartApiResponse] object
+     * @param response : HTTP response received from the API
+     * @return SignUpStartApiResponse object
+     */
+    @Throws(ClientException::class)
+    fun getSignUpStartResultFromHttpResponse(
+        response: HttpResponse
+    ): SignUpStartApiResponse {
+        LogSession.logMethodCall(TAG, "${TAG}.getSignUpStartResultFromHttpResponse")
+
+        val result = if (response.body.isNullOrBlank()) {
+            SignUpStartApiResponse(
+                response.statusCode,
+                DEFAULT_ERROR,
```

Comment:
Not sure where default error is defined or what it means but you may want a specific and meaningful error that indicates the exact issue

Reply:
I have changed it to `EMPTY_RESPONSE_ERROR`.